amr-wind/wind_energy/actuator/FLLC.H

```diff
@@ -33,6 +34,16 @@ struct FLLCData
     bool different_sizes;
     RealList span_distance_vel;
     RealList span_distance_force;
+
+    // non-uniform variables
+    bool nonuniform{true};      // non-uniform flag
```

Comment:
I think it is best to leave the original/default behavior. 
It would help in testing/debugging and it is consistent with the original derivations in the manuscript. 
If the ALM data structure had the non-uniform distribution, then we would not need to use this non-uniform option and it could default to the old behavior.